src/style.css

```diff
@@ -1,3 +1,88 @@
+* {
+  /* Скидання параметрів браузера */
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  display: flex;
+}
+
+
+
+
+.logo-1200 {
```

Comment:
The class '.logo-1200' seems to be named based on style or size, which is not recommended. Class names should represent the meaning of the content, not the styles or dimensions.